.github/workflows/regression.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Regression Testing
+
+on:
+  deployment_status:
```

Comment:
Isn't this `deployment_status` what caused the infinite loop of actions?

Reply:
It appears to be the combination of on: deployment_status: and on: push that did it. Its currently out there right now with on deployment status and it isnt misbehaving. Also theres no craziness happening in the Actions tab either with these pushes.

Reply:
The ones out there right now are disabled. But do we even need this file if only running local anyway?

Reply:
Removed here: 517ffaf9fcf5e76929e9409a8e8dca027af75ac5